### ISSUE TITLE
Is it possible to implement weighted frame loss?

### ISSUE THREAD

**Issue by KinWaiCheuk** [jongwook/onsets-and-frames#13]
You mentioned that you did not implement the weighted frame loss in this repository. 
But I am curious on how to do it with PyTorch. Do you have any insights?

**Comment by jongwook** [comment]
You'll first need to construct the appropriate weight matrix, with the same size as the label, and then supply it as the optional `weight` parameter of `F.binary_cross_entropy`.

See [the documentation](https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.binary_cross_entropy) for more usage.
